doc/api.rst

```diff
@@ -57,7 +57,10 @@ reads are represented as :class:`~pysam.PileupRead` objects in the
 
     import pysam
     samfile = pysam.AlignmentFile("ex1.bam", "rb" )
-    for pileupcolumn in samfile.pileup("chr1", 100, 120):
+    
+    pileup = bamfile.pileup("chr1", 100, 120)
```

Comment:
This surely isn't what you meant. After the change, you're creating a variable called `samfile` above, but referencing `bamfile` here.